xrdp/xrdp.ini.in

```diff
@@ -217,7 +217,7 @@ port=-1
 #chansrvport=DISPLAY(0)
 
 [vnc-any]
-name=vnc-any
+name=VNC Proxy
```

Comment:
Please don't change this one - existing user will be using it.

Reply:
I will revert.
However, I don't think "vnc-any" and "neutrinordp-any" can be associated with "Proxy".
So instead, I want to add a ; comment .
Something like the following

`name=vnc-any
; VNC Proxy`